src/solver/model_getters.jl

```diff
@@ -24,6 +24,16 @@ function get_model_config(parsed_args)
     end
 end
 
+function get_sea_sfc_temp(parsed_args)
```

Comment:
```suggestion
function get_sea_sfc_temp(parsed_args)
```
```suggestion
function get_sfc_temperature(parsed_args)
```
@charleskawczynski Any idea for a better name? This is not really getting the surface temperature, it is just getting the functional form (symmetric or asymmetric).

Reply:
`sfc_temperature_form`?

Reply:
Sounds good to me, thanks!